src/lib/components/Assessment/index.js

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {html} from "lit-element";
+import {BaseModalElement} from "../BaseModalElement";
+
+/**
+ * Element that renders a self-assessment callout.
+ * @extends {BaseModalElement}
+ * @final
+ */
+class Assessment extends BaseModalElement {
+  static get properties() {
+    return {
+      modal: {attribute: "aria-modal", reflect: true},
```

Comment:
Why is `this.modal` different than `this.open`?

Reply:
This is to toggle the `aria-modal` property when the modal is open. Normally, you'd leave it on all the time, but since the assessment is moving itself, it shouldn't be left on when not in the modal state, I believe. (This is super weird. We shouldn't be moving this thing around, as noted in the outstanding issues. I think @robdodson has an idea for how to do this better.)

Reply:
After considering how much markup could be copied if/when you get more complex question types (a lot!), I decided to stick with (an improved version of) the moving approach. So I think we still need to toggle `aria-modal` when the assessment is visible in its unopened state (i.e., on desktop).